docs/enterprise.rst

```diff
@@ -0,0 +1,10 @@
+For Enterprise
```

Comment:
FYI you can include that chunk from README using sphinx.

Reply:
Can I? I'm not aware of how to include a section of another file in a file. If I could, that would be great. 

_does the minimal amount of research_

I do see [this example](https://muffinresearch.co.uk/selectively-including-parts-readme-rst-in-your-docs/) which leads me to the `start-after` and `end-before` directives in [the docs](http://docutils.sourceforge.net/docs/ref/rst/directives.html#including-an-external-document-fragment).

If I could include a section by name, I think I'd implement that, but since I can't and I must bound the included content with start/stop sentinels, I'm more inclined to just live with the duplication for now.

Reply:
Yes, that's the one. Use `include` or `literalinclude`. The trick is to use comment lines which won't be rendered anyway. You could also add some `-do-not-remove-` thing into those comments.

Reply:
In dca4a517, I added a "todo" so as not to forget.